ingestion-beam/src/test/java/com/mozilla/telemetry/DecoderMainTest.java

```diff
@@ -15,7 +15,7 @@
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
 
-public class DecoderMainTest {
+public class DecoderMainTest extends Decoder {
```

Comment:
In a few places, I make a test class extend a class whose constructor otherwise would never get called, and thus would show up as uncovered.

Reply:
wouldn't it be better to have a test that calls the constructor?

Reply:
Yeah, that would be more explicit. Will do.